Makefile

```diff
@@ -367,7 +367,7 @@ simplestreams: juju juju-metadata ${SIMPLESTREAMS_TARGETS}
 	@echo "\nRun export JUJU_METADATA_SOURCE=\"${JUJU_METADATA_SOURCE}\" if not defined in your env"
 
 .PHONY: build
-build: rebuild-schema go-build
+build: rebuild-schema rebuild-client-schema go-build
```

Comment:
I'm of the mind to make client the default for `rebuild-schema`.

Reply:
This would be simpler! I wasn't sure if we needed the full schemas for anything else, but it sounds like python-libjuju is the only user of the schemas. Assuming we iron out whether we need any other schemas (controller schemas?). In conversation with the Australia Juju team members this morning, they said that probably if there are any non-client schemas that python-libjuju needs then they should be added to the client schemas.